examples/howto/plot_batch_simulate.py

```diff
@@ -68,10 +68,68 @@ def set_params(param_values, net=None):
     'sigma': np.linspace(1, 20, 5)
 }
 
+###############################################################################
+# Define a function to calculate summary statistics
+
+
+def summary_func(results):
+    """
+    Calculate the min and max dipole peak for each simulation result.
+
+    Parameters
+    ----------
+    results : list
+        List of dictionaries containing simulation results.
+
+    Returns
+    -------
+    summary_stats : list
+        Summary statistics for each simulation result.
+    """
+    summary_stats = []
+    for result in results:
+        dpl_data = result['dpl'][0].data['agg']
+        min_peak = np.min(dpl_data)
+        max_peak = np.max(dpl_data)
+        summary_stats.append({'min_peak': min_peak, 'max_peak': max_peak})
+    return summary_stats
+
 ###############################################################################
 # Run the batch simulation and collect the results.
 
-batch_simulation = BatchSimulate(set_params=set_params)
+# Comment off this code, if dask and distributed Python packages are installed
+# from dask.distributed import Client
+# client = Client(threads_per_worker=1, n_workers=5, processes=False)
+
+
+# Run the batch simulation and collect the results.
+batch_simulation = BatchSimulate(set_params=set_params,
+                                 summary_func=summary_func)
 simulation_results = batch_simulation.run(param_grid,
                                           n_jobs=n_jobs,
-                                          combinations=False)
+                                          combinations=False,
+                                          backend='multiprocessing')
+# backend='dask' if installed
+print("Simulation results:", simulation_results)
+###############################################################################
+# Extract and Plot Results
```

Comment:
In the next PR let's add the ability to return different types of simulation data, as well as specify which ones should be saved

Reply:
Similarly we should be able to return the parameter values